### ISSUE TITLE
big loop is not supported now

### ISSUE THREAD

**Issue by XhinLiang** [mvel/mvel#234]
If eval a expression like `list = []; for (int i = 0; i < 521; i++) { list.add(i) }` will cause
```
Caused by: java.lang.NoSuchMethodException: ASMAccessorImpl_80404262315883800703666.<init>(org.mvel2.compiler.ExecutableStatement)
        at java.lang.Class.getConstructor0(Class.java:3082)
        at java.lang.Class.getConstructor(Class.java:1825)
        at org.mvel2.optimizers.impl.asm.ASMAccessorOptimizer._initializeAccessor(ASMAccessorOptimizer.java:768)
        at org.mvel2.optimizers.impl.asm.ASMAccessorOptimizer.compileAccessor(ASMAccessorOptimizer.java:895)
        ... 20 common frames omitted
2020-05-02 08:41:10 [as-command-execute-daemon] WARN  c.t.a.c.command.klass100.MvelCommand -mvel: failed execute express: list = []; for (int i = 0; i < 521; i++) { list.add(i) }
```

but eval `list = []; for (int i = 0; i < 5; i++) { list.add(i) }` will be OK.

**Comment by MalcolmOdd** [comment]
Hi @XhinLiang ,
I tried and could not reproduce this with 2.4.7.Final. A few bugs in ASMAccessOptimizer were fixed in that release. Can you confirm which release you're using and provide a unit test if possible?
Otherwise a workaround would be to disable ASMAccessOptimizer by setting mvel2.disable.jit to true or calling  OptimizerFactory.setDefaultOptimizer("reflective").

**Comment by XhinLiang** [comment]
Hi @MalcolmOdd .

Sorry but I can not reproduce this in unit test even in lower version like `2.4.4.Final`, but in my repo [](https://github.com/XhinLiang/arthas-mvel) it just happened, both `2.4.7.Final` and `2.4.4.Final`, I can not find where the problem came from.

Thankfully I try `OptimizerFactory.setDefaultOptimizer("reflective")` and it fixed.

**Comment by XhinLiang** [comment]
Additionally, my code in reop arthas-mvel usually run in `Java Instrumentation`, is that the reason of this Exception?

Anyway, `OptimizerFactory.setDefaultOptimizer("reflective")` works for me now, thank you!

**Comment by MalcolmOdd** [comment]
Glad to hear it. OptimizerFactory.setDefaultOptimizer("reflective") is a workaround and may have performance drawbacks since it prevents MVEL from switching over to compiled mode when an expression is evaluated repeatedly. Depending on your needs this might be acceptable.
Your code base uses a 13 months old copy of mvel (https://github.com/XhinLiang/jugg/commits/master/src/main/java/org/mvel2) so you're missing fixes like #219 which might be relevant in this case. I suggest closing this issue since it has no actionable item in the main code base.

**Comment by XhinLiang** [comment]
arthas-mvel is using official mvel now.
